### PR TITLE
Created properties

### PR DESCRIPTION
**Pull request by YuraMitriakhin**

### CODE REVIEW

File: src/main/java/com/gmail/yuramitryahin/config/AppConfig.java [hunk 1]
```diff
@@ -4,22 +4,29 @@
...
+    private Environment env;
+
     @Bean
     public DataSource getDataSource() {
         BasicDataSource dataSource = new BasicDataSource();
-        dataSource.setDriverClassName("com.mysql.cj.jdbc.Driver");
-        dataSource.setUrl("jdbc:mysql://localhost:3306/spring?serverTimezone=UTC");
-        dataSource.setUsername("root");
-        dataSource.setPassword("2801");
+        dataSource.setDriverClassName(env.getProperty("db.driver"));
```
Comment (IwalkInSilence) [inline src/main/java/com/gmail/yuramitryahin/config/AppConfig.java]: mb rename on environment ?

**Review by IwalkInSilence — APPROVED** [review]

File: src/main/java/com/gmail/yuramitryahin/config/AppConfig.java [hunk 2]
```diff
@@ -4,22 +4,29 @@
...
+import org.springframework.core.env.Environment;
 import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
 
 @Configuration
+@PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {"com.gmail.yuramitryahin.dao",
         "com.gmail.yuramitryahin.service"})
 public class AppConfig {
+    @Autowired
+    private Environment env;
```
Comment (Sofasmile) [inline src/main/java/com/gmail/yuramitryahin/config/AppConfig.java]: What is better: setter, constructor, or field injection? Think why.

Comment (Sofasmile) [inline src/main/java/com/gmail/yuramitryahin/config/AppConfig.java]:
```suggestion
    private Environment environment;
```

**Review by Sofasmile — CHANGES_REQUESTED** [review]

File: src/main/java/com/gmail/yuramitryahin/config/AppConfig.java (hunk 2 above)
Comment (YuraMitriakhin) [reply src/main/java/com/gmail/yuramitryahin/config/AppConfig.java]: constructor

Comment (YuraMitriakhin) [reply src/main/java/com/gmail/yuramitryahin/config/AppConfig.java]: fixed

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**